woocommerce/woocommerce-android
# Issue/4864 jetpack benefits bottom banner

Closes: #4864 


### Description
This PR adds a bottom banner that allows checking the Jetpack benefits when the current site doesn't have full Jetpack, it can be dismissed by the user, and will be re-displayed after 5 days from dismissal date.

Clicking on the banner will be handled in a separate PR.

### Testing instructions
1. You need a site that has Jetpack CP for this test.
2. Open the app, and select the Jetpack CP site.
3. Open MyStore, and confirm that you can see the bottom banner.
4. Click on the "X" button to dismiss it.
5. Close the app and open it again, and confirm that it's not getting displayed again.
6. Change the time of your device to advance it by 5 days or more.
7. Open the app again, and confirm that you can see the bottom banner now.

Please test using a regular site, and confirm that the banner is never displayed for it.

### Images/gif
| Light | Dark |
| --- | --- |
| <img width="733" alt="Screen Shot 2021-10-13 at 12 31 30" src="https://user-images.githubusercontent.com/1657201/137124945-f6f1778d-49e4-4b8a-bb3c-27bfd7f9247a.png"> |<img width="733" alt="Screen Shot 2021-10-13 at 12 31 50" src="https://user-images.githubusercontent.com/1657201/137124958-ed8cb209-f39a-45a1-bbec-3b5536d8ef2e.png"> |
